subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1352,6 +1352,59 @@ static int cmd_wifi_ap_sta_disconnect(const struct shell *sh, size_t argc,
 	return 0;
 }
 
+static int cmd_wifi_ap_config(const struct shell *sh, size_t argc,
+			      char *argv[])
+{
+	struct net_if *iface = net_if_get_first_wifi();
+	struct wifi_ap_config param = { 0 };
+	long val = 0;
+	int err = 0;
+	int ret = -1;
+	int idx = 1;
+	bool param_valid = false;
+
+	context.sh = sh;
+
+	while (idx < argc) {
+		if (!strncasecmp(argv[idx], "max_inactivity", 14)) {
+			val = shell_strtol(argv[++idx], 10, &err);
+			if (err) {
+				shell_error(sh, "Unable to parse input (err %d)", err);
+				return err;
+			}
+			param.ap_max_inactivity = (uint32_t)val;
+			param.type = WIFI_AP_CONFIG_PARAM_MAX_INACTIVITY;
+			param_valid = true;
+		}
+		if (!strncasecmp(argv[idx], "max_num_sta", 11)) {
```

Comment:
```suggestion
		else if (!strncasecmp(argv[idx], "max_num_sta", 11)) {
```

Reply:
I missed that you are using a single shell command to configure all params, in which case please either use separate individual commands or move to `getopt` like we do for `connect`, `ap_enable` etc that can take multiple params.